src/CacheWarmer/EntrypointCacheWarmer.php

```diff
@@ -22,7 +22,7 @@ class EntrypointCacheWarmer extends AbstractPhpFileCacheWarmer
     public function __construct(array $cacheKeys, string $phpArrayFile, CacheItemPoolInterface $fallbackPool)
```

Comment:
Not sure if we can drop the `$fallbackPool` as it would be a BC break

Reply:
I think it's ok to remove it

Reply:
Ok I'll do it then ;)

Reply:
Done